src/styles/_base_layout.scss

```diff
@@ -2,56 +2,39 @@
 // ======================================
 
 // Container
-.container {
-  @include screen-md-min {
-    max-width: $screen-md - $gutter-width;
-  }
-  @include screen-lg-min {
-    max-width: $screen-lg - $gutter-width;
-  }
-  @include screen-xl-min {
-    max-width: $screen-xl - $gutter-width;
-  }
-
-  position: relative;
-  width: 100%;
-  max-height: 100%;
-  margin: 0 auto;
-  padding-right: ($gutter-width / 2);
-  padding-left: ($gutter-width / 2);
+[class*="__container"] {
```

Comment:
would suggest applying these individually, as tedious as that may be -> helps to maintain all relevant styles to a section to a single location

Reply:
just use the mixin you've written on each one!